sopel/modules/reddit.py

```diff
@@ -238,6 +288,12 @@ def auto_redditor_info(bot, trigger, match):
     redditor_info(bot, trigger, match)
 
 
+@url(subreddit_url)
+def auto_subreddit_info(bot, trigger, match):
+    match = str(trigger).split("/")[-1]
+    subreddit_info(bot, trigger, match)
```

Comment:
You have `match` already… Why this? Just do `subreddit_info(bot, trigger, match.group(1))`.